### PR TITLE
Update docs for CosmWasm with Testnet

### PR DESCRIPTION
**Pull request by Yoon-Suji**
## What is the purpose of the change
This pull request improves documation of area `cosmwasm-testnet` by adding contents to deploying and executing CosmWasm smart contract on osmosis testnet.

## Brief change log
- Added content in `cosmwasm-testnet.md`

## Verifying this change
This change has been tested locally by rebuilding the doc website and verified content and links are expected

### CODE REVIEW

**Comment by sigridjineth** [comment]
LGTM 👍 glad to your first contribution to web3 field

**Review by sigridjineth — APPROVED** [review]

**Review by sigridjineth — APPROVED** [review]
LGTM 👍 glad that your first contribution to web3/crypto field seems great!

File: developing/dapps/get_started/cosmwasm-testnet.md [hunk 1]
```diff
@@ -1,4 +1,321 @@
...
+
+**Important** write this mnemonic phrase in a safe place.
+It is the only way to recover your account if you ever forget your password.
+
+divert cliff issue spirit penalty chief improve neck enjoy pipe sing loop inherit behind space next tourist acid axis easy never ball enemy moment
+```
+
+You need some tokens named `OSMO`(`uosmo`) in your address to interact with the network.
+
+You must participate in the [Osmosis discode](https://discord.com/invite/osmosis) to request a faucet of the Osmosis Testnet. After gaining access to the testnet channel on the `#roles` channel of the discode, you can request a testnet token by sending the following message on the `#faucet` channel:
```
Comment (xBalbinus) [inline developing/dapps/get_started/cosmwasm-testnet.md:91]:
```suggestion
You must participate in the [Osmosis discord](https://discord.com/invite/osmosis) to request a faucet of the Osmosis Testnet. After gaining access to the testnet channel on the `#roles` channel of the discord, you can request a testnet token by sending the following message on the `#faucet` channel:
```

File: developing/dapps/get_started/cosmwasm-testnet.md [hunk 2]
```diff
@@ -1,4 +1,321 @@
...
+
+- `osmosisd query bank balances [address]` commands query the total balance of an account.
+- `osmosisd keys show -a wallet` commands returns the address of the wallet that you created.
+
+
+
+## Deploy a Smart Contract
+### Clone cw-template
+
+For this example, we will use the [**cw-template**](https://github.com/InterWasm/cw-template) (CosmWasm Template) repo with counter example.
```
Comment (xBalbinus) [inline developing/dapps/get_started/cosmwasm-testnet.md:114]: Osmosis actually has a cw_template https://github.com/osmosis-labs/cw-tpl-osmosis for you to use... the deploy process is also in the docs.

File: developing/dapps/get_started/cosmwasm-testnet.md [hunk 3]
```diff
@@ -1,4 +1,321 @@
...
+
+```bash
+# Linux
+sudo apt-get install jq
+
+# Mac
+brew install jq
+```
+
+ If you are using an operating system other than the one listed above, please refer to the [official document](https://stedolan.github.io/jq/download/).
```
Comment (xBalbinus) [inline developing/dapps/get_started/cosmwasm-testnet.md:198]:
```suggestion
If you are using an operating system other than the one listed above, please refer to the [official document](https://stedolan.github.io/jq/download/).
```

**Review by xBalbinus — CHANGES_REQUESTED** [review]
Thanks so much for your great work. This all mostly looks good, see comments for some minor feedback.

**Comment by Yoon-Suji** [comment]
@xBalbinus Thank you for your kind comment! I made a new commit based on your comments. please check🙏

**Comment by daniel-farina** [comment]
@Yoon-Suji @sigridjineth 
Docs are live: https://docs.osmosis.zone/developing/dapps/get_started/cosmwasm-testnet-manual.html

Thank you so much

**Pull request closed without merging**